packages/@okta/vuepress-theme-prose/components/MenuItem.vue

```diff
@@ -12,6 +12,7 @@
       :href="item.link"
       :class="{ [itemCss]: true, active: $page.path.includes(item.link) }"
       :target="item.target"
+      :rel="item.target ? 'noopener noreferrer': undefined"
```

Comment:
Just to be sure, will vue set `rel="undefined"` or will it just omit the property?

Reply:
hmm, I think we can try something like that `:rel="item.target && 'noopener noreferrer`, In case of undefined rel attr should be removed